From: Simon McVittie <smcv@debian.org>
Date: Sat, 31 Aug 2024 01:08:21 +0100
Subject: testsuite: Skip instead of failing if /dev/udmabuf is inaccessible

Not all kernels even have this, and on those that have it, an
autobuilder might not have access.

Bug: https://gitlab.gnome.org/GNOME/gtk/-/issues/6971
Forwarded: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/7671
Signed-off-by: Simon McVittie <smcv@debian.org>
---
 testsuite/gdk/dmabuf-support.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/gdk/dmabuf-support.c b/testsuite/gdk/dmabuf-support.c
index 738b7ba..89e4c7e 100644
--- a/testsuite/gdk/dmabuf-support.c
+++ b/testsuite/gdk/dmabuf-support.c
@@ -17,7 +17,7 @@ test_dmabuf_no_gpu (void)
 
   if (!udmabuf_initialize (&error))
     {
-      g_test_fail_printf ("%s", error->message);
+      g_test_skip_printf ("%s", error->message);
       g_error_free (error);
       return;
     }
